### ISSUE TITLE
dont force rewrite the locastorage

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#2849]

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}
//set the version type
process.env.VERS = "patch"

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci $VERS`);

// types only package
exec('npm config set //registry.npmjs.org/:_authToken $NPM_TOKEN')
exec(`cd ./types-only && npm version $VERS && npm run build && npm publish && cd ..`);
exec(`git commit -a -m 'updated types-only package'`);
exec(`git push --force`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [4.44.4](https://github.com/open-wa/wa-automate-nodejs/compare/4.44.3...4.44.4) (2022-08-24)

- Release 4.44.4 [`5634a78`](https://github.com/open-wa/wa-automate-nodejs/commit/5634a78069fae09f049533c35de0e137e2e1e081)
- ✨ CLI: `-c/--config` can now be path #2845 [`622ac27`](https://github.com/open-wa/wa-automate-nodejs/commit/622ac2702090aedf5b6185f222761987f95d5381)
- ✨ CLI: `cli.config.js` default export function #2844 [`d6f5085`](https://github.com/open-wa/wa-automate-nodejs/commit/d6f508581a73daf271e66690790625e29130e5c7)
- 🔧 only overwrite localstorage if `legacy` is `true` #2849 [`54eef43`](https://github.com/open-wa/wa-automate-nodejs/commit/54eef4322c354f76cf5cf6ec81182b457c91c6ff)
- ✏️ fix typo in cli options [`5ce8e77`](https://github.com/open-wa/wa-automate-nodejs/commit/5ce8e77070de3750ad45d69822ab2287c28dedd0)
- 🥚 patches update [`1c6bd1e`](https://github.com/open-wa/wa-automate-nodejs/commit/1c6bd1e61d126a44dadfa2f4e44dd0bb12fd02f8)
- updated types-only package [`ea0c2fb`](https://github.com/open-wa/wa-automate-nodejs/commit/ea0c2fb2b28e7649b2376e3217def0378e904850)
